package.json

```diff
@@ -2,7 +2,21 @@
   "name": "mantra-demo",
   "version": "0.0.0",
   "dependencies": {
-  
+  },
+  "eslintConfig": {
+    "extends": "eslint:recommended",
```

Comment:
Cool. I am gonna write tests and the testing setup today.

I will leave this for you :)
On 2016 ජන 9, සෙන at පෙ.ව. 7.36 Maxime Quandalle <notifications@github.com>
wrote:

> In package.json
> <https://github.com/kadira-samples/mantra-demo/pull/2#discussion_r49258443>
> :
>
> > @@ -2,7 +2,21 @@
> >    "name": "mantra-demo",
> >    "version": "0.0.0",
> >    "dependencies": {
> > -
> > +  },
> > +  "eslintConfig": {
> > +    "extends": "eslint:recommended",
>
> Will do that, and the migration to beta-3 tomorrow (unless you do it
> before).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kadira-samples/mantra-demo/pull/2/files#r49258443>.
>